plugins/apitracing/src/lib/ConstantDefinitions.h

```diff
@@ -0,0 +1,19 @@
+#ifndef APITRACING_CONSTANTDEFINITIONS_H
+#define APITRACING_CONSTANTDEFINITIONS_H
+
+#include <cstdint>
+
+namespace ApiTracing
+{
+    class ConstantDefinitions
```

Comment:
Why do you need a class for that? Imho it's misused in this context. Did you mean to create a subnamespace?

Reply:
I think there was a lint that this was discouraged. But I can't reproduce this now. So a subnamespace it is. :-D